lib/modules/userInfo.js

```diff
@@ -101,61 +96,142 @@ module.options = {
 export const highlightedUsers = {};
 
 module.go = () => {
-	if (module.options.hoverInfo.value) {
-		$(document.body).on('mouseover', UserTagger.usernameSelector, handleMouseOver);
-	}
+	observe(document.body, { childList: true }, mutation => {
+		if (!mutation.addedNodes.length) return;
+		// author-tooltip
+		const addedNode = downcast(mutation.addedNodes[0], HTMLElement);
+		if (addedNode.classList.contains('author-tooltip')) {
+			setTimeout(() => renderHoverCardInfo(addedNode), 50);
```

Comment:
Can be avoided with
```
waitForChild(addedNode, '.author-tooltip__profile-link')
  .then(link => {
     const [, username] = UserTagger.usernameRE.exec(authorLink.getAttribute('href'));
     applyResData(addedNode, username);
  })
);
```

Reply:
Doesn't `waitForChild` only work for direct children? (the `author-tooltip__profile-link` is fairly nested)

Reply:
You're right. A similar function which observes with option `subtree: true` would have to be added.